content/en/news/releases/1.6.x/announcing-1.6/_index.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Announcing Istio 1.6
+linktitle: 1.6
+subtitle: Major Update
+description: Istio 1.6 release announcement.
+publishdate: 2020-05-21
+release: 1.6.0
+skip_list: true
+aliases:
+    - /news/announcing-1.6.0
+    - /news/announcing-1.6
+---
+
+We are pleased to announce the release of Istio 1.6!
+
+{{< relnote >}}
+
+With this release, we are continuing the path we charted earlier this year in
+our [roadmap post](/blog/2020/tradewinds-2020/), sailing toward more
+simplicity, a better installation experience, and some other goodies as well.
+
+Here’s some of what’s coming to you in today's release:
+
+## Simplify, simplify, simplify
+
+Last release, we introduced **Istiod**, a component that reduced the number of
+components in an Istio installation by combining the functionality of several
+other components. In Istio 1.6, we have completed this transition and have fully
+moved functionality into Istiod. This has allowed us to remove Citadel, the
+sidecar injector, and Galley.
+
+Great news! We've got a simplified experience for developers who are taking
```

Comment:
This is only Alpha in Kubernetes 1.18 and so sounds a little overselling to me.

What about "As Kubernetes advanced and adds support for defining protocols, Istio will be able to learn from this too. In the upcoming 1.19 release, support for defining an `AppProtocol` on a `Service` will mean you will no longer need to append the name field  to denote the protocol.

Better link is also https://github.com/kubernetes/enhancements/issues/1507

Reply:
I'll take your link and rephrase to include alpha. But I'm also going to make it clear that Kubernetes has this in alpha in 1.18.